ENG-70744 - forked upstream to tweak logged content and to add Debian…

andrus801 opened the pull request:
log content tweaks and debian packaging

andrus801 committed:
ENG-70744 - forked upstream to tweak logged content and to add Debian packaging.

sdouglas-skytap commented on bashrc-patch:
```diff
@@ -1,80 +1,70 @@
 }
 
+# set HISTCONTROL
+HISTCONTROL="ignoredups"
```
Why would we want to ignoredups? Would it be useful to know if a command was run more than once?

sdouglas-skytap commented on debian/copyright:
```diff
@@ -0,0 +1,15 @@
+Source: https://github.com/skytap/shell-session-logger
+
+Files: *
+Copyright: 2020 Skytap, Inc
```
Should this be attributed to the original owner?

andrus801 replied:
Updated based on conversation w/ original owner.

andrus801 committed:
ENG-70744: Update PR
bashrc-patch - simplify and improve HISTCONTROL handling
debian/copyright - fix naming and add section for upstream content
(based on communication with author)

andrus801 committed:
ENG-70744: PR feedback + other cleanup
- change package name to add "skytap" prefix
- fix permission on sudoers include to recomened 0440
- fix dh_installdocs issue on ubuntu 16.04

andrus801 committed:
ENG-70744: back out 'skytap-' namespacing

andrus801 committed:
ENG-70744: tidy up README.md
